app/models/repository/git.rb

```diff
@@ -222,6 +225,15 @@ def priority_settings
     @priority_settings ||= OpenStruct.new(Settings.git[:push_priority])
   end
 
+  def mark_ontology_as_having_file(path, has_file)
```

Comment:
Maybe use the keyword-syntax for `has_file` as this shows better what the boolean stands for in calls. e.g.
`mark_ontology_as_having_file(path, has_file: true)` is more readable than `mark_ontology_as_having_file(path, true)`.